modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/processor/AstArtifactConfigurationProcessorTestCase.java

```diff
@@ -34,6 +34,7 @@
 import org.mule.runtime.deployment.model.api.artifact.ArtifactContext;
 import org.mule.runtime.deployment.model.api.artifact.ArtifactContextConfiguration;
 import org.mule.runtime.extension.api.dsl.syntax.resources.spi.ExtensionSchemaGenerator;
+import org.mule.runtime.module.deployment.internal.AbstractMuleWithTestLoggingSupportTestCase;
 import org.mule.tck.junit4.AbstractMuleTestCase;
```

Comment:
This import seems no longer necessary. Please check. If so, please also correct it in all the other test cases that were updated.